action.yml

```diff
@@ -43,9 +43,9 @@ inputs:
       Only applicable when 'mount-docker-socket' is true.
     required: false
     default: /var/run/docker.sock
-  docker-cmd-file:
```

Comment:
please add `docker-cmd-file` as deprecated input for now but don't document in readme. So we're backward compatible